Bottom: fe019460fb4cbbdd3bda3409d8216e5eb3ce62ab
Top:    552506d5ade617503aa842566b9452581dec6ac2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-21 02:45:04 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
